dev/integration_tests/flutter_gallery/lib/demo/shrine/supplemental/product_card.dart

```diff
@@ -52,7 +52,7 @@ class ProductCard extends StatelessWidget {
                 child: imageWidget,
               ),
               SizedBox(
-                height: kTextBoxHeight * MediaQuery.of(context).textScaleFactor,
+                height: kTextBoxHeight * MediaQuery.textScalerOf(context).textScaleFactor, // ignore: deprecated_member_use
```

Comment:
Are there plans to clean this up? Can you link an issue: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#comment-all-ignores

(here and everywhere else)

Reply:
Same question for all the places that were migrated to TextScaler.textScaleFactor.

(I am also wondering what the point of migrating everything to the also deprecated `TextScaler.textScaleFactor` API is. Why not leave it as the old deprecated API and clean it up properly by migrating to the real TextScaler when we're ready for that?)

Reply:
Added a link to #128825. As for git history since I'll have to add a temporary `ignore` anyways I'll keep the intermediate form for now.